contrib/format-excel/src/main/java/org/apache/drill/exec/store/excel/ExcelBatchReader.java

```diff
@@ -363,6 +407,22 @@ private void setFirstRow() {
     }
   }
 
+  /**
+   * This function is used to set the iterator to the first row of actual data.  When a schema is provided,
+   * we can safely skip the header row, and start reading the first row of data.
+   */
+  private void setFirstDataRow() {
+    // Initialize
+    currentRow = rowIterator.next();
```

Comment:
Is it possible to do this (?) : 
```java
// 1. Remove this line
// 2. Add a loop once
for (int i = 0; i <= rowNumber + 1; i++) {
  currentRow = rowIterator.next();
}
```

Reply:
I suppose so.  Not sure what the advantage is though.

Reply:
The logic has not changed and is easy to understand. Because at least two iterations are required.